client/my-sites/comments/comment/index.jsx

```diff
@@ -42,8 +42,16 @@ export class Comment extends Component {
 	};
 
 	componentWillReceiveProps( nextProps ) {
-		if ( nextProps.isBulkMode && ! this.props.isBulkMode ) {
-			this.setState( { isExpanded: false } );
+		if ( nextProps.isPostView ) {
+			if ( ! this.props.isPostView ) {
```

Comment:
you can express this `if` as:

```js
if ( ! this.props.isPostView || this.props.isBulkMode !== nextProps.isBulkMode ) {
  this.setState( { isExpanded: false } );
}
```

or even shorter:
```js
componentWillReceiveProps( nextProps ) {
	const isExpanded = nextProps.isPostView
		? ! ( ! this.props.isPostView || this.props.isBulkMode !== nextProps.isBulkMode )
		: ! nextProps.isLoading && ! nextProps.isBulkMode;
	this.setState( { isExpanded } )
}
```

Reply:
I actually chose to be as clear as possible to avoid confusion on the logic.
If folks think a shorter approach is as clear or even more than my code, I'll happily update it! :)

Reply:
Actually, the ternary's first expression could end up setting `isExpanded = true`, which is not what we want.
My brain is melting on this thing and I'm gonna push a `let` solution. Sorry in advance! 😄

Reply:
Maybe this component has too many `is` props. But don't stress too much about this. I just need to use the same approach on the Comment View, that's why I was revisiting this.